mozilla-lockbox/lockbox-android
# Make entire username and password views clickable

Fixes #281
Depends on #399

To aim to be atomic, and since these changes are in _slightly_ different territory, this adds on to the commit at #382 #399.

## Testing and Review Notes

1. open item detail view
2. tap to confirm the entire username/password row is 'clickable'
3. (accessibility) confirm the voice over instructions explain the row will "copy your username/password"

Note: because the text area appears on screen before the "reveal password" button, it may be confusing, but the accessibility / tab order is intentional:

1. web address (open in browser)
2. username (copy)
3. password (copy)
4. password (reveal)


## Screenshots or Videos

![android_emulator_-_pixel_2_api_28_5554](https://user-images.githubusercontent.com/49511/51637757-d9c94300-1f19-11e9-8649-1816ac54531b.png)



## To Do

- add “WIP” to the PR title if pushing up but not complete nor ready for review
  - optional: consider adding instrumentation (integration/UI) tests
- consider running this branch in a debug simulator and check for memory leak notifications or any warnings
- [x] fix the "password is always revealed" logic bug I introduced
- [ ] put the "null input" in a better way/right place? if applicable...
